### PR TITLE
[docs] Improvements for Start tab [skip ci]

### PR DESCRIPTION
**Pull request by stasadev**
I took a closer look at the Start tab in the new documentation https://ddev.readthedocs.io/en/latest/ and changed a few things:

* fixed several small typos
* corrected the position for `!!!warning`, `!!!note`, `!!!tip` (they must be separated by a new line from everything above)
* removed extra spaces
* fixed code style for bash inserts
* removed/changed links from the old docs
* "macOS Full Disk Access for Special Directories" was in Windows tab
* "macOS-specific NFS debugging" was in the wrong place (not in the macOS tab, but under it, and with broken formatting)
* added formatting for file paths (e.g. settings.php became `settings.php`)

<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/3979"></a>

### CODE REVIEW

**Comment by rfay** [comment]
Awesome, thanks! I just enabled pull requests on readthedocs, https://docs.readthedocs.io/en/stable/pull-requests.html - so you should be able to push another commit and see the results on readthedocs.

**Comment by rfay** [comment]
Opened https://github.com/readthedocs/readthedocs.org/issues/9422

File: docs/content/users/quickstart.md [hunk 1]
```diff
@@ -359,30 +359,30 @@ Here's a quickstart instructions for a number of different environments:
...
+**Note:** If you're providing the `settings.php` or `wp-config.php` and DDEV is creating the `settings.ddev.php` (or `wp-config-local.php`, `AdditionalConfig.php`, or similar), the main settings file must explicitly include the appropriate DDEV-generated settings file.  Any changes you need should be included somewhere that loads after DDEV's settings file, for example in Drupal's `settings.php` *after* `settings.ddev.php` is included. (see [Adding Configuration](#adding-configuration) below).
 
-**Note:** If you do *not* want DDEV-Local to create or manage settings files, set `disable_settings_management: true` in your .ddev/config.yaml or `ddev config --disable-settings-management` and you will be the only one that edits or updates settings files.
+**Note:** If you do *not* want DDEV-Local to create or manage settings files, set `disable_settings_management: true` in your `.ddev/config.yaml` or `ddev config --disable-settings-management` and you will be the only one that edits or updates settings files.
```
Comment (rfay) [inline docs/content/users/quickstart.md:364]: This should probably be an admonition right?
```suggestion
!!!note "Turning off settings management completely"

    If you do *not* want DDEV-Local to create or manage settings files, set `disable_settings_management: true` in your `.ddev/config.yaml` or `ddev config --disable-settings-management` and you will be the only one that edits or updates settings files.
```

File: docs/content/users/quickstart.md [hunk 2]
```diff
@@ -359,30 +359,30 @@ Here's a quickstart instructions for a number of different environments:
...
 
 The `ddev config` command attempts to create a CMS-specific settings file with DDEV credentials pre-populated.
```
Comment (rfay) [inline docs/content/users/quickstart.md:366]: I suspect the following should be tabbed?

**Review by rfay — CHANGES_REQUESTED** [review]
Absolutely great work, thanks! I made a couple of suggestions you can just click the button to commit if you like, and 
* the suggested admonition probably needs looking at when running `mkdocs serve`
* We have to find out why the spellchecker didn't flag the egregious typos.
* If you want to take on tabbing the extra-config section please add that. If not, it can wait. 

Thanks for the great work! I haven't run it under mkdocs yet. Hopefully we can also get the readthedocs PR display working. 

If you're willing to try it, please rebase since https://github.com/drud/ddev/pull/3980 went in. The only problem is that you don't have easy access to mkdocs-material-insiders, but I think it only breaks search for you.

File: docs/content/users/quickstart.md (hunk 2 above)
Comment (stasadev) [reply docs/content/users/quickstart.md:368]: Looks good.

image

**Comment by stasadev** [comment]
> Thanks for the great work!

You are welcome!

> If you're willing to try it, please rebase since #3980 went in.

Done.

> the suggested admonition probably needs looking at when running `mkdocs serve`

Done.

> If you want to take on tabbing the extra-config section please add that. If not, it can wait.

I'm busy with other things now.

> We have to find out why the spellchecker didn't flag the egregious typos.

I also noticed (locally with my spellchecker) that any spell checking rules don't apply to lines that aren't at the root level. Literally just indent and every check is lost.

**Comment by rfay** [comment]
I think your theory is correct about spellchecker. Opened
* https://github.com/drud/ddev/issues/3981
Sorry you had to see all those typos! There are plenty more, will have to sort that.


Thanks so much for doing this!

**Pull request merged**